README.md

```diff
@@ -43,6 +43,9 @@ So if you run multiple AFL++ instances on your fuzzing target, you can disable C
 This [document](https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/fuzzing_in_depth.md)
 will familiarize you with AFL++ features to help in running a successful fuzzing campaign.
 
+By default, 'fuzzing' config is set when `cargo-afl` is used to build. If you want to prevent this, just set 'AFL_NO_CFG_FUZZING=1`
+environmental variable, when building.
```

Comment:
Thanks,, @lrubasze!

Just one nit. Could this please read something like the following?
> set the environment variable `AFL_NO_CFG_FUZZING` to `1` when building.

Reply:
My english really sucks 😄  Thanks for spotting this.